mysql-test/t/audit_log_acc.test

```diff
@@ -0,0 +1,71 @@
+# Bug 1641910: Trying to set audit_log_exclude_accounts crashes server.
```

Comment:
This test case does a lot of server restarts to test how different variables combinations are tested on startup. We don't have similar test cases currently.